libbeat/docs/shared-ssl-config.asciidoc

```diff
@@ -211,157 +340,225 @@ key: |
 ----
 
 [float]
-==== `key_passphrase`
-
-The passphrase used to decrypt an encrypted key stored in the configured `key` file.
-
-[float]
-==== `supported_protocols`
-
-List of allowed SSL/TLS versions. If SSL/TLS server decides for protocol versions
-not configured, the connection will be dropped during or after the handshake. The
-setting is a list of allowed protocol versions:
-`SSLv3`, `TLSv1` for TLS version 1.0, `TLSv1.0`, `TLSv1.1`, `TLSv1.2`, and
-`TLSv1.3`.
-
-The default value is `[TLSv1.1, TLSv1.2, TLSv1.3]`.
-
-[float]
+[[client-verification-mode]]
 ==== `verification_mode`
 
-Controls the verification of certificates. Valid values are:
+Controls the verification of server certificates. Valid values are:
 
- * `full`, which verifies that the provided certificate is signed by a trusted
+`full`::
+Verifies that the provided certificate is signed by a trusted
 authority (CA) and also verifies that the server's hostname (or IP address)
 matches the names identified within the certificate.
- * `strict`, which verifies that the provided certificate is signed by a trusted
+
+`strict`::
+Verifies that the provided certificate is signed by a trusted
 authority (CA) and also verifies that the server's hostname (or IP address)
 matches the names identified within the certificate. If the Subject Alternative
 Name is empty, it returns an error.
- * `certificate`, which verifies that the provided certificate is signed by a
+
+`certificate`::
+Verifies that the provided certificate is signed by a
 trusted authority (CA), but does not perform any hostname verification.
- * `none`, which performs _no verification_ of the server's certificate. This
+
+`none`::
+Performs _no verification_ of the server's certificate. This
 mode disables many of the security benefits of SSL/TLS and should only be used
 after very careful consideration. It is primarily intended as a temporary
 diagnostic mechanism when attempting to resolve TLS errors; its use in
 production environments is strongly discouraged.
-
++
 The default value is `full`.
 
-[float]
-==== `cipher_suites`
+[discrete]
+[[ssl-server-config]]
+=== Server configuration options
 
-The list of cipher suites to use. The first entry has the highest priority.
-If this option is omitted, the Go crypto library's https://golang.org/pkg/crypto/tls/[default suites]
-are used (recommended). Note that TLS 1.3 cipher suites are not
-individually configurable in Go, so they are not included in this list.
+In the +{beatname_lc}.yml+ config file, you can specify the following server options in the
+`ssl` section of each output that supports SSL.
```

Comment:
We have no outputs that would act as server. Some inputs/metricsets though.

Reply:
To avoid any confusion, I made this more generic.